client/components/edit/create/__snapshots__/CreateObject.test.tsx.snap

```diff
@@ -67,11 +67,15 @@ exports[`CreateObject renders appropriately with editing disabled 1`] = `
       value=""
     />
   </label>
-  <input
-    name="parent"
-    type="hidden"
-    value="foo:1234"
-  />
+  <label>
+    Parent ID
+    <input
```

Comment:
If editing is disabled, I think this should render as a hidden element rather than a text input. Did something change?

Reply:
I didn't see the need to use a hidden element, since that value is already stored in a react state.

Reply:
Good point regarding the lack of need for a hidden input... but in any case, the intention is that the visible parent element should not display when parent editing is disabled. If that became visible unintentionally as a result of refactoring, further adjustment may be needed. (I haven't done extensive hands-on testing around this particular issue, though, so maybe I'm misreading the changes in the snapshots... I can spend time on that if you need me to).